compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -542,6 +542,12 @@ error.service.object.type.required=\
 error.service.invalid.object.type=\
   given type ''{0}'' does not match with service type interface
 
+error.service.function.invalid.modifier=\
+  function defined in services cannot have an explicit visibility qualifier
+
+error.service.function.invalid.invocation=\
+  service functions cannot be invoked outside service descriptor
```

Comment:
Is this error message okay?

Reply:
How about "invalid service function invocation"?

Reply:
Fixed.